docs/governance/rfc/002-go-cors-authn.md

```diff
@@ -0,0 +1,56 @@
+# 002: CORS and authn for Go
+
+This RFC proposes adding two ancillary Go projects to the Connect Github
+organization:
+
+* `connectrpc.com/cors`, which eases server-side CORS configuration, and
+* `connectrpc.com/authn`, which provides flexible server-side authentication
+  middleware.
+
+## CORS
+
+[Cross-origin resource sharing][cors], commonly called CORS, allows servers to
+indicate whether web browsers should allow scripts to load resources from other
+domains. This is a common concern for single-page web applications, where the
+API and app are often hosted on different origins &mdash; for example,
+`app.acme.com` may need to call APIs hosted on `api.acme.com`. As part of its
+security model, CORS requires servers to allowlist individual request and
```

Comment:
Small suggestion but we should try to replace `allowlist` as a verb here - I think CORS requires servers to send additional HTTP request/response headers (and support preflight requests but that wouldn't affect Connect that uses GET/POST).

Reply:
I'll reword. I didn't want to get into the exact CORS flow, but in practice Connect servers need to respond to preflight requests with request and response header allowlist. I'll clarify the text.